server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -206,3 +206,19 @@ body {
 main {
   flex: 1 0 auto;
 }
+
+// Specify size of program cards in a grid
+@media (min-width: 55em) {
+  .tablet-lg\:grid-col-6 {
+    flex: 0 1 auto !important;
+    width: 50% !important;
```

Comment:
question (blocking): I'm confused about the need to set the width explicitly. Shouldn't using `grid` handle how many elements fit in a row?

Reply:
Good point. I rewrote this part, what do you think?